llvm/tools/llvm-objcopy/ObjcopyOpts.td

```diff
@@ -230,3 +230,15 @@ defm add_symbol
 defm update_section
     : Eq<"update-section", "Replace the contents of section <name> with contents from a file <file>">,
       MetaVarName<"name=file">;
+
+defm gap_fill
+    : Eq<"gap-fill", "Fill the gaps between sections with <value> instead of zero. "
+                     "<value> must be an unsigned 8-bit integer. "
+                     "This option is only supported for ELF inputs and binary outputs.">,
+      MetaVarName<"value">;
+
+defm pad_to
+    : Eq<"pad-to", "Pad the output to the load address <address>, using a value "
+                   "of zero or the value specified by :option:`--gap-fill`"
+                   "This option is only supported for ELF inputs and binary outputs.">,
```

Comment:
Help message typically omit the trailing period

Reply:
Done. I also removed formatting, not sure why I added it here.